src/main/java/com/azure/cosmos/examples/crudquickstart/async/SampleCRUDQuickstartAsync.java

```diff
@@ -0,0 +1,342 @@
+
+import java.time.Duration;
+import java.util.concurrent.CountDownLatch;
+import java.util.stream.Collectors;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class SampleCRUDQuickstartAsync {
+
+    private CosmosAsyncClient client;
+
+    private final String databaseName = "AzureSampleFamilyDB";
+    private final String containerName = "FamilyContainer";
+
+    private CosmosAsyncDatabase database;
+    private CosmosAsyncContainer container;
+
+    protected static Logger logger = LoggerFactory.getLogger(SampleChangeFeedProcessor.class.getSimpleName());
+
+    public void close() {
+        client.close();
+    }
+
+    /**
+     * Run a Hello CosmosDB console application.
+     *
+     * @param args command line args.
+     */
+    //  <Main>
+    public static void main(String[] args) {
+        SampleCRUDQuickstartAsync p = new SampleCRUDQuickstartAsync();
+
+        try {
+            System.out.println("Starting ASYNC main");
+            p.getStartedDemo();
+            System.out.println("Demo complete, please hold while resources are released");
+        } catch (Exception e) {
+            e.printStackTrace();
+            System.err.println(String.format("Cosmos getStarted failed with %s", e));
+        } finally {
+            System.out.println("Closing the client");
+            p.shutdown();
+        }
+    }
+
+    //  </Main>
+
+    private void getStartedDemo() throws Exception {
+        //This is a simple sample application intended to demonstrate Create, Read, Update, Delete (CRUD) operations
+        //with Azure Cosmos DB Java SDK, as applied to databases, containers and items. This sample will
+        //1. Create asynchronous client, database and container instances
+        //2. Create several items
+        //3. Upsert one of the items
+        //4. Perform a query over the items
+        //5. Delete an item
+        //6. Delete the Cosmos DB database and container resources and close the client.
+
+        System.out.println("Using Azure Cosmos DB endpoint: " + AccountSettings.HOST);
+
+        ConnectionPolicy defaultPolicy = ConnectionPolicy.getDefaultPolicy();
+        //  Setting the preferred location to Cosmos DB Account region
+        //  West US is just an example. User should set preferred location to the Cosmos DB region closest to the application
+        defaultPolicy.setPreferredLocations(Lists.newArrayList("West US"));
+
+        //  Create async client
+        //  <CreateAsyncClient>
+        client = new CosmosClientBuilder()
+            .setEndpoint(AccountSettings.HOST)
+            .setKey(AccountSettings.MASTER_KEY)
+            .setConnectionPolicy(defaultPolicy)
+            .setConsistencyLevel(ConsistencyLevel.EVENTUAL)
+            .buildAsyncClient();
+
+        //  </CreateAsyncClient>
+
+        createDatabaseIfNotExists();
+        createContainerIfNotExists();
+
+        Family andersenFamilyItem=Families.getAndersenFamilyItem();
```

Comment:
code style formatting:
```java
Family andersenFamilyItem=Families.getAndersenFamilyItem();
```
change to 
```
Family andersenFamilyItem = Families.getAndersenFamilyItem();
```

please use intellij auto formatting on the sample files.

Reply:
Fixed in next commit.